components/brave_wallet/browser/brave_wallet_constants.h

```diff
@@ -27,51 +27,93 @@ constexpr int32_t kAutoLockMinutesMin = 1;
+      true, false, "YFI", 18, true, "", "", mojom::kMainnetChainId},
      {"0x9043d4d51C9d2e31e3F169de4551E416970c27Ef", "Palm DAI", "pdai.png",
-      true, false, "PDAI", 18, true, "", "", ""}});
+      true, false, "PDAI", 18, true, "", "", mojom::kMainnetChainId}});
+
+// List of assets from Ramp, available to buy
+static base::NoDestructor<std::vector<mojom::BlockchainToken>> kRampBuyTokens(
+    {{"", "Ethereum", "", false, false, "ETH", 18, true, "", "",
+      mojom::kMainnetChainId},
+     {"", "Avalanche", "", false, false, "AVAX", 18, true, "", "",
+      mojom::kAvalancheMainnetChainId},
+     {"0x82030cdbd9e4b7c5bb0b811a61da6360d69449cc", "RealFevr", "", true, false,
+      "BSC_FEVR", 18, true, "", "", mojom::kBinanceSmartChainMainnetChainId},
+     {"", "Celo", "", false, false, "CELO", 18, true, "", "",
+      mojom::kCeloMainnetChainId},
+     {"0xD8763CBa276a3738E6DE85b4b3bF5FDed6D6cA73", "Celo Euro", "ceur.png",
+      true, false, "CEUR", 18, true, "", "", mojom::kCeloMainnetChainId},
+     {"0x765DE816845861e75A25fCA122bb6898B8B1282a", "Celo Dollar", "cusd.png",
+      true, false, "CUSD", 18, true, "", "", mojom::kCeloMainnetChainId},
+     {"0x6b175474e89094c44da98b954eedeac495271d0f", "DAI Stablecoin", "dai.png",
+      true, false, "DAI", 18, true, "", "", mojom::kMainnetChainId},
+     {"", "Polygon", "", false, false, "MATIC", 18, true, "", "",
+      mojom::kPolygonMainnetChainId},
+     {"0x8f3cf7ad23cd3cadbd9735aff958023239c6a063", "DAI Stablecoin", "dai.png",
+      true, false, "MATIC_DAI", 18, true, "", "",
+      mojom::kPolygonMainnetChainId},
+     {"0x7ceB23fD6bC0adD59E62ac25578270cFf1b9f619", "Ethereum", "eth.png", true,
+      false, "MATIC_ETH", 18, true, "", "", mojom::kPolygonMainnetChainId},
+     {"0xbbba073c31bf03b8acf7c28ef0738decf3695683", "Sandbox", "sand.png", true,
+      false, "MATIC_SAND", 18, true, "", "", mojom::kPolygonMainnetChainId},
+     {"0x2791bca1f2de4661ed88a30c99a7a9449aa84174", "USD Coin", "usdc.png",
+      true, false, "MATIC_USDC", 6, true, "", "",
+      mojom::kPolygonMainnetChainId},
+     {"", "Solana", "", false, false, "SOLANA_SOL", 9, true, "", "",
+      mojom::kSolanaMainnet},
+     {"Es9vMFrzaCERmJfrF4H2FYD4KCoNkY11McCe8BenwNYB", "Tether", "usdt.png",
+      false, false, "SOLANA_USDT", 6, true, "", "", mojom::kSolanaMainnet},
+     {"0xdac17f958d2ee523a2206206994597c13d831ec7", "Tether", "usdt.png", true,
+      false, "USDT", 6, true, "", "", mojom::kMainnetChainId},
+     {"0xa0b86991c6218b36c1d19d4a2e9eb0ce3606eb48", "USD Coin", "usdc.png",
+      true, false, "USDC", 6, true, "", "", mojom::kMainnetChainId}});
 
 const char kWalletBaseDirectory[] = "BraveWallet";
 const char kImageSourceHost[] = "erc-token-images";
 const char kWyreID[] = "AC_MGNVBGHPA9T";
-const char kBuyUrl[] =
+const char kWyreBuyUrl[] =
     "https://pay.sendwyre.com/?dest=ethereum:%s"
     "&destCurrency=%s&amount=%s&accountId=%s&paymentMethod=debit-card";
+const char kRampBuyUrl[] =
```

Comment:
@muliswilliam I believe we need some kind of API key or identifier unique to Brave. Can you please confirm? Also the config query parameters are different from the ones you had in your PR.

Reply:
@onyb I am using [Hosted Mode](https://docs.ramp.network/configuration) which does not required an API key. The query params are documented on the above link. Which ones are different?

Reply:
We do want Brave to earn commissions through onramps via Ramp, right? If so, how do we indicate whether a transaction was done on the Brave Wallet?

Reply:
@onyb I will get this and use it.

Reply:
Ok let's add that in your frontend PR.